tests/parm/rasmgshoc.nml.IN

```diff
@@ -0,0 +1,355 @@
+&amip_interp_nml
+       interp_oi_sst = .true.
+       use_ncep_sst = .true.
+       use_ncep_ice = .false.
+       no_anom_sst = .false.
+       data_set = 'reynolds_oi',
+       date_out_of_range = 'climo',
+/
```

Comment:
This namelist `amip_interp_nml` is not used in ufs-weather-model (fv3atm). At some point (maybe even in this PR) we should remove it from all input namelist (templates). Few other namelists like `coupler_nml` and `interpolator_nml`
 should also be removed.